.travis.yml

```diff
@@ -69,6 +69,12 @@ install:
   - export APP_CHANGED=False;
   - for entry in `git diff --name-only HEAD~1`; do if [[ "$entry" == "keras/applications/"* ]]; then export APP_CHANGED=True; fi; done
 
+  # set utf-8
+  - export LC_ALL=C.UTF-8;
+  - export LANG=C.UTF-8;
```

Comment:
It seems that travis already uses UTF8 by default. See https://docs.travis-ci.com/user/environment-variables/#default-environment-variables .

I don't think that those two lines are useful.

Reply:
Thanks for reviewing! I checked and removed it.

Reply:
Thanks!